app/page.tsx

```diff
@@ -19,6 +20,131 @@ export default function Home() {
             consectetur adipiscing elit, sed du eiusmod tempor.
           </Paragraph>
         </div>
+        <div className="max-w-prose">
+          <Heading
+            as="h2"
+            size={6}
+            leading="normal"
+            className="rs-mb-2 rs-mt-8 "
+          >
+            Some of our{' '}
+            <Text
+              as="span"
+              color="black-true"
+              bgColor="purple"
+              className="px-10 rounded"
+            >
+              custom
+            </Text>{' '}
+            highlights.
+          </Heading>
+          <Paragraph>
+            Building websites that fit the custom needs of our clients across
+            Stanford, employing Storyblok, Lorem Ipsum, and our very own,
+            Stanford Sites.
+          </Paragraph>
+          <div className="h-400">
+            <Paragraph>Cards coming soon...</Paragraph>
+          </div>
+        </div>
+        <div className="">
+          <Heading
+            as="h2"
+            size={6}
+            leading="normal"
+            className="rs-mb-2 rs-mt-8"
+          >
+            Service in the{' '}
+            <Text as="span" bgColor="teal" className="px-10 rounded">
+              numbers.
+            </Text>
+          </Heading>
+          <Grid md={4} gap="default">
+            <Card icon={'user'} paddingType="small" iconColor="blue">
+              <Heading>66/34</Heading>
+              <Paragraph className="text-black-40 big-paragraph font-semibold">
+                New clients/ returning clients
+              </Paragraph>
+            </Card>
+            <Card icon={'badge'} paddingType="small" iconColor="purple">
+              <Heading>1,456</Heading>
+              <Paragraph className="text-black-40 big-paragraph font-semibold">
+                Resolved SNOW tickets
+              </Paragraph>
+            </Card>
+            <Card icon={'clock'} paddingType="small" iconColor="illuminating">
```

Comment:
is there a reason to pass the icon prop in like this` icon={'clock'}` instead of just `icon="clock"`? 👀